### DISCUSSION TITLE
Limit Maximum Retry

### DISCUSSION THREAD

**Discussion by SaimonL** [net-ssh/net-ssh#930] [Q&A]
Hi is there a way to limit maximum retry `Net::SSH.start` will do?
I have some hosts that are sometimes unreachable and I would like to not retry if the connection fails.
At the moment it auto retry 2 times before giving up.

Thanks

**Comment by vicomte** [comment]
I limit just by connection timeout. Unless the host's connection is very unstable I rarely find that retrying a failed connection once will make it better.
